.travis/Brewfile.travis

```diff
@@ -0,0 +1,8 @@
+# using https://github.com/Homebrew/homebrew-bundle/blob/master/README.md#usage as example
+brew "sdl"
+brew "sdl_gfx"
+brew "sdl_mixer"
+brew "sdl_ttf"
+brew "glew" unless system "[[ \"$HYPERROGUE_USE_GLEW\" != \"1\" ]]"
+brew "libpng" unless system "[[ \"$HYPERROGUE_USE_PNG\" != \"1\" ]]"
+brew "automake" unless system "[[ \"$TRAVIS_USE_AUTOTOOLS\" != \"1\" ]]"
```

Comment:
How is this Brewfile thing better than what came before? It seems like this is just the same list of manual `brew install` commands, except that now they're obfuscated by missing the `install` part of the `brew install`, and they're off in this new directory (which again is _not HyperRogue_ 😛). Does this Brewfile business help TravisCI cache things better or something? Do you see a speedup in build times?

Reply:
Yeah, using `Brewfile` may be a bit overkill, however it's the recommended way of using Homebrew [per Travis' docs](https://docs.travis-ci.com/user/reference/osx/#homebrew). Well, the addon is, and the `bundle` deal specifically is because of the need for conditional installation.

I only really did the switch in hopes that it would work around `brew update` issues, which it didn't. So this one is liable to get rolled back as well.